.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/redux-async
```

Comment:
This should probably be something like 'avocado' or something?

Reply:
changed 🥑